src/annotator/guest.js

```diff
@@ -538,41 +521,6 @@ export default class Guest extends Delegator {
     return this.createAnnotation({ $highlight: true });
   }
 
-  // Create a blank comment (AKA "page note")
-  createComment() {
```

Comment:
This method was never used in app code. The "Page Note" button in the sidebar's vertical toolbar calls `createAnnotation` and either creates an annotation or note depending on whether there is a selection.